autovmaf_generate_excel.py

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env python3
+from __future__ import annotations
```

Comment:
Needed for EP 563 – Postponed Evaluation of Annotations https://docs.python.org/3/library/__future__.html